gateway/src/apicast/proxy.lua

```diff
@@ -85,6 +85,8 @@ local function output_debug_headers(service, usage, credentials)
     ngx.header["X-3scale-credentials"]   = credentials
     ngx.header["X-3scale-usage"]         = usage
     ngx.header["X-3scale-hostname"]      = ngx.var.hostname
+    ngx.header["X-3scale-svc-id"]        = service.id
```

Comment:
Why abbreviating to `svc` instead of using `service`?

Reply:
Just for readability X-3scale-service-name looked a really long name.

Reply:
@palmieric It is just one character more than `X-3scale-credentials`. I'd prefer `service` instead of `svc`.

Reply:
Ok will change it and resubmit

Reply:
You can just force push to this PR.

And you'll need a changelog entry.

Reply:
Done

Reply:
You'll still need to add a new entry to CHANGELOG.md file.